Automattic/jetpack
# Admin Menu: Remove style dependencies

Fixes https://github.com/Automattic/wp-calypso/issues/51535.
Counterpart of 659-gh-Automattic/wpcomsh (hotfix for Atomic sites).

#### Changes proposed in this Pull Request:
Removes the masterbar styles dependencies from the admin menu styles. They are unnecessary, and was causing some nav unification issues when they were missing (like when [AMP is enabled](https://github.com/Automattic/jetpack/blob/f8e877deb5f5b74f3e99bfabdd2603c683231998/projects/plugins/jetpack/modules/masterbar/masterbar/class-masterbar.php#L138-L144)).

#### Jetpack product discussion
N/A.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
_Simple sites_
* Apply the diff below to your WP.com sandbox.
* Sandbox a WP.com simple site.
* Visit a WP Admin of the sandboxed site.
* Make sure there are no visual regressions in the unified menu.

_Atomic sites_
* Install and activate the Jetpack Beta plugin.
* Go to Jetpack > Jetpack Beta, and switch to the branch of this PR.
* Install and activate the AMP plugin.
* Go to AMP > Settings and activate the "Standard" mode.
* Make sure there are no visual regressions in the unified menu.
